README.md

```diff
@@ -50,49 +50,41 @@ Note that depending on the plugin you add, there might be required permissions a
 repositories {
   mavenCentral()
   maven {
-    url 'https://api.mapbox.com/downloads/v2/releases/maven'
-    authentication {
-      basic(BasicAuthentication)
-    }
-    credentials {
-      username "mapbox"
-      password = "SDK_REGISTRY_TOKEN"
-    }
+    url 'https://api.maplibre.com/downloads/v2/releases/maven'
```

Comment:
This does not exist.

Reply:
:+1:, the artifacts are on maven central (see https://mvnrepository.com/artifact/org.maplibre.gl/android-plugin-annotation-v9), so this `maven` block can be removed

Reply:
@tempranova If you can take care of this then we can remove it.